Index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <div class="mobile-container">
+      <header class="topnav">
+        <nav>
+          <a href="#home" class="active">My Logo</a>
+          <a class="icon"><img src="images/btn.png" /></a>
+        </nav>
+      </header>
+      <section>
+        <main>
+          <h1 class="headline">Hey There. I’m Yaser.</h1>
+          <p class="title">I'm a Software Developer</p>
```

Comment:
- [optional] please, all the complete title should be in the h1 tag. Kindly adjust that.